python/pyspark/pandas/tests/test_series.py

```diff
@@ -1161,13 +1161,29 @@ def test_append(self):
     def test_map(self):
         pser = pd.Series(["cat", "dog", None, "rabbit"])
         psser = ps.from_pandas(pser)
+
+        # dict correspondence
         # Currently Koalas doesn't return NaN as pandas does.
```

Comment:
nit: Not related with this patch, but would you fix `Koalas` -> `pandas API on Spark` while we're here ?

Reply:
Good catch!! Modified.